model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -97,7 +97,9 @@ class ReverseChannelsPropagationDown(BackReplacementPattern):
         'BatchNormalization': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_eltwise(node, rc),
         'FakeQuantize': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_eltwise(node, rc),
         'Multiply': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_eltwise(node, rc),
+        'Divide': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_eltwise(node, rc),
         'Add': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_eltwise(node, rc),
+        'Subtract': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_eltwise(node, rc),
         'Pow': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_eltwise(node, rc),
         'Convert': lambda node, rc: ReverseChannelsPropagationDown.pass_rc_through_eltwise(node, rc),
 
```

Comment:
What is it about `Maximum`, `Minimum`, `SquaredDifference`? Also I checked the specification and didn't observe `Pow`. Maybe it needs to be changed to `Power`?

Reply:
These nodes don't appear on the paths where we pull/push/transition reverse input channel node. So I assume not to include them here.

Reply:
We didn't explore such models so far but we should not exclude such cases. It looks a potential future bug.